src/components/Main/AppMenu.tsx

```diff
@@ -1,41 +1,40 @@
-import React from 'react'
-import { Menu } from 'semantic-ui-react'
-import { Link, RouteComponentProps, withRouter } from 'react-router-dom'
+import React, { useState } from 'react'
+import { Header, Menu, MenuProps } from 'semantic-ui-react'
+import { RouteComponentProps, withRouter } from 'react-router-dom'
 import { useRouter } from '../../utils/hooks/useRouter'
-import { useUserState } from '../../contexts/UserState'
+import { TemplatesDetails } from '../../utils/types'
 
 interface AppMenuProps extends RouteComponentProps {
-  items: Array<Array<String>>
+  templatePermissions: TemplatesDetails
 }
 
-const AppMenu: React.FC<AppMenuProps> = (props: AppMenuProps) => {
-  const { pathname } = useRouter()
-  const {
-    userState: { templatePermissions },
-  } = useUserState()
+// THIS IS NOT WORKING YET - Code is in UserArea. To be moved here.
```

Comment:
Okay, I'm ignoring this file for now. Any comments are in UserArea.tsx

Reply:
Comment removed. Now it's using the AppMenu and Links got fixed, the UserArea needed to be inside the `Router` so moved to inside `SiteLayout` container.